app/components/rdfa-editor-container.hbs

```diff
@@ -104,19 +104,6 @@
                 {{yield to='sidebar'}}
                 {{#if @shouldEditRdfa}}
```

Comment:
My bad on not clarifying the ticket, but I already put in a mechanism to hide the rdfa editing tools, this boolean. It should really just be a matter of setting that boolean to false in `templates/regulatory-statements/edit.hbs`
however, see other comment